Library/Homebrew/formula_installer.rb

```diff
@@ -251,6 +251,7 @@ def install
         opoo "Bottle installation failed: building from source."
         raise BuildToolsError, [formula] unless DevelopmentTools.installed?
       else
+        puts @requirement_messages
```

Comment:
Make a new function with `puts @requirement_messages` like `def puts_requirement_messages` or similar which outputs nothing i.e. no newline which `@requirement_messages` is empty